src/components/MultiSelectQuestion.tsx

```diff
@@ -4,7 +4,7 @@ import { SubmitQuestion } from "./SubmitQuestion";
 
 interface QuestionProps {
   questionNum: string;
-  question: string;
+  question: string | JSX.Element;
   answers: string[];
   onSubmit: (answers: string[]) => void;
```

Comment:
Ideally this interface should be recycled between MultiselectQuestion, RadioQuestion, and TextQuestion instead of being copied across the files. You can change onSubmit's type to (answer: string)=>void, and convert the array of answers from multiselect into a comma-separated string (i.e. turn ["A", "B"] into "A,B") to make the types work across all three classes.